internal/statemachine/classic_states.go

```diff
@@ -287,6 +281,51 @@ func (stateMachine *StateMachine) createChroot() error {
 	return nil
 }
 
+// add PPAs to the apt sources list
+func (stateMachine *StateMachine) addExtraPPAs() error {
+	var classicStateMachine *ClassicStateMachine
+	classicStateMachine = stateMachine.parent.(*ClassicStateMachine)
+
+	// create /etc/apt/sources.list.d in the chroot if it doesn't already exist
+	sourcesListD := filepath.Join(stateMachine.tempDirs.chroot, "etc", "apt", "sources.list.d")
```

Comment:
FWIW this directory is owned by the apt package. If apt isn't installed, this code is going to have a bad time :) I guess this is really here for tests?

Reply:
Yes this is really there for the tests :)